proposals/csharp-7.1/async-main.md

```diff
@@ -17,7 +17,8 @@ It is very common when learning C#, when writing console-based utilities, and wh
 to call and `await` `async` methods from Main.  Today we add a level of complexity here by forcing such `await`'ing to be
 done in a separate async method, which causes developers to need to write boilerplate like the following just to get
 started:
-```C#
+
+```csharp
```

Comment:
just curious, does this make a difference?

Reply:
It depends on the markdown engine. `csharp` is supported more widely.